csrc/mamba/mamba_ssm/selective_scan_fwd.cu

```diff
@@ -311,26 +311,21 @@ void selective_scan_fwd_launch(SSMParamsBase &params, cudaStream_t stream) {
     // Only kNRows == 1 is tested for now, which ofc doesn't differ from previously when we had each block
     // processing 1 row.
     constexpr int kNRows = 1;
+    constexpr bool kIsVariableB = true;
+    constexpr bool kIsVariableC = true;
+    constexpr bool kHasZ = true;
```

Comment:
Could you add torch checks to guard against cases where `kIsVariableB`, `kIsVariableC`, or `kHasZ` is false?

Reply:
Done